requirements.txt

```diff
@@ -1,4 +1,5 @@
 -r lint-requirements.txt
+ansible>=2.5,<3
```

Comment:
I'm not sure whether we want to set the higher limit + Ansible doesn't follow semver: the change in the second number in version designates a major change. So 2.6 and 2.7 are major versions while 2.6.1 would be minor and so on.

I think for apps like Molecule it's more sustainable in the long run to use an open higher limit.

Reply:
So I'd just do
```suggestion
ansible>=2.5
```

Reply:
Fair enough, will amend!